dom.bs

```diff
@@ -3591,6 +3592,9 @@ run these steps:
 <var>target</var> with null, null, null, <var>addedNodes</var>, <var>removedNodes</var>,
 <var>previousSibling</var>, and <var>nextSibling</var>.
 
+<p class="note no-backref">Either <var>addedNodes</var> or <var>removedNodes</var>
+<a for=list>is not empty</a>.
```

Comment:
This seems nice to call out, but then we should turn the thing above this into an algorithm with two steps and the first step is this, as an assert.

Reply:
Resolved in de10bc3.